### PR TITLE
Replace BaseFieldDescription::camelize with Inflector::camelize

### PR DESCRIPTION
**Pull request by afurculita**
This moves the camelize responsibilities to Inflector

### CODE REVIEW

**Comment by OskarStark** [comment]
Hmm 

I Think its better to keep the camelize method. So it could be overwritten

But use the inflector IN this method should be done

**Comment by afurculita** [comment]
@OskarStark I don't really see a case where you need to overwrite the camelize function. Using the `camelize` function from `Inflector` and not allowing to be overwritten means code more consistent throughout the project.

File: Admin/AdminHelper.php [hunk 1]
```diff
@@ -184,18 +184,4 @@ public function addNewInstance($object, FieldDescriptionInterface $fieldDescript
...
-    /**
-     * Camelize a string.
-     *
-     * @static
-     *
-     * @param string $property
-     *
-     * @return string
-     */
-    public function camelize($property)
```
Comment (Soullivaneuh) [inline Admin/AdminHelper.php]: This is BC break.

If this method is really not needed, it should be deprecated instead.

File: Admin/BaseFieldDescription.php [hunk 2]
```diff
@@ -415,20 +415,6 @@ public function getMappingType()
...
     /**
-     * Camelize a string.
-     *
-     * @static
-     *
-     * @param string $property
-     *
-     * @return string
-     */
-    public static function camelize($property)
```
Comment (Soullivaneuh) [inline Admin/BaseFieldDescription.php]: This is BC break.

If this method is really not needed, it should be deprecated instead.

File: Tests/Admin/BaseFieldDescriptionTest.php [hunk 3]
```diff
@@ -194,13 +193,6 @@ public function testGetTranslationDomain()
         $this->assertEquals('ExtensionDomain', $description->getTranslationDomain());
     }
 
-    public function testCamelize()
```
Comment (Soullivaneuh) [inline Tests/Admin/BaseFieldDescriptionTest.php]: This should be kept for BC.

**Comment by Soullivaneuh** [comment]
Just a question: Why choosing Doctrine Inflector instead of Symfony camelizer?

**Comment by Soullivaneuh** [comment]
Also tests need to be fixed.

File: Tests/Admin/BaseFieldDescriptionTest.php (hunk 3 above)
Comment (rande) [inline Tests/Admin/BaseFieldDescriptionTest.php]: Yes, methods should behave the same way

**Comment by rande** [comment]
ok to test

**Comment by Soullivaneuh** [comment]
Closing since the forked repository seems to not exist anymore.

**Pull request closed without merging**